### ISSUE TITLE
pepco - wikidata id redirected |  checking  redirected wikidata id-s ..

### ISSUE THREAD

**Issue by ImreSamu** [osmlab/name-suggestion-index#3065]
TLDR:  we need to check the wikidata redirections 

https://nsi.guide/index.html?k=shop&v=clothes#  searching for "pepco"    expecting 'shop/clothes|Pepco'

#  Now  
Potomac Electric Power Company   "supplier of electric power to Washington, D.C., and its Maryland suburbs"
but this is not a shop=clothes company

![image](https://user-images.githubusercontent.com/2217320/66158468-3c84e980-e626-11e9-8fbf-4500239ed5d8.png)

# Expecting: 

original website:  https://pepco.pl/  
logo:
![image](https://user-images.githubusercontent.com/2217320/66159327-0cd6e100-e628-11e9-9b72-3236b7553e99.png)

# Debugging 

debugging :  Q11815580   Redirected to:  Q7166207  : https://www.wikidata.org/w/index.php?title=Q11815580&action=history

**Comment by bhousel** [comment]
> TLDR: we need to check the wikidata redirections

huh, I did not know that was a thing  🤔

**Comment by ImreSamu** [comment]
quick fix:  I have redirected to the correct wikidata id. 
*  Q11815580  ->  [Q65470183](https://www.wikidata.org/wiki/Q65470183)   "one of the largest non-food retail chains in Central and Eastern Europe"

audit log:  https://www.wikidata.org/w/index.php?title=Q11815580&action=history

**Comment by bhousel** [comment]
@ImreSamu I'm still really confused by this..
Wasn't Q11815580 the correct QID?
Can't we just keep using it?

The other Pepco is https://www.wikidata.org/wiki/Q7166207
There was never a conflict.

**Comment by ImreSamu** [comment]
@bhousel : 

>Wasn't Q11815580 the correct QID?
>Can't we just keep using it?

first:  I have tried to revert the redirections and keep the  Q11815580  ;
but the wikidata system not allowed this . ..  
  *  because some wikipedia pages ( https://pl.wikipedia.org/wiki/Pepco ) linked to new  wikidata   ;
so the only easy fix:  redirecting the new - correct  wikidata id.

my understanding : 
* we added a new complexity with the wikidata id -s :)  

the good new:
* https://taginfo.openstreetmap.org/tags/wikidata=Q11815580  - is not used .. strange ..

**Comment by ImreSamu** [comment]
testing with iD editor  ( hungarian interface  )
- the logo is not correct   ( maybe some caching issues )
- the wikidata label correct ( it is from the redirection )  - but not in hungarian  ( the hungarian label is exists - I have added to the wikidata 10 minutes before )
- using wikidata=Q11815580 as in the metafile  ( expected )

![image](https://user-images.githubusercontent.com/2217320/66163201-0ba9b200-e630-11e9-83cc-928d83860173.png)

**Comment by bhousel** [comment]
> first: I have tried to revert the redirections and keep the Q11815580 ;
> but the wikidata system not allowed this . ..

I'm pretty sure a Wikidata admin can sort this out.. It would be kind of annoying to change all of the wikidata QIDs in OSM to match this new number - not really that complicated with JOSM or level0, but it would be nice to avoid an unnecessary change.

I'll reach out to the Wikidata folks on IRC and see if they can help us out.

**Comment by 1ec5** [comment]
I’ve [reverted Q11815580](https://www.wikidata.org/wiki/Special:Diff/1025469457) to the last version by @bhousel and merged Q65470183 back into it. (I had to manually move the interwiki links first to avoid a conflict.) As far as I can tell, [the changes to Q7166207](https://www.wikidata.org/wiki/Special:Diff/970328804) had already been undone manually at some point. Long story short, Q11815580 is the QID for this chain, Q65470183 redirects to it, and no changes are needed in this repository or OSM.
